2025.html

```diff
@@ -89,7 +89,8 @@ <h3>Kudos to these websites who <!-- TODO: got --> plan to get naked in 2025!</h
             <li><a href="https://tommi.space/">tommi.space</a></li>
             <li><a href="https://villepreux.net/">Antoine Villepreux</a></li>
             <li><a href="https://waby.ink/">waby.ink</a></li>
-            <li><a href="https://silvanzwick.com/">Silvan Zwick</a></li>
+            <li><a href="https://silvanzwickr.com/">Silvan Zwick</a></li>
+            <li><a href="https://gofreerange.com/">Go Free Range</a></li>
```

Comment:
Would you mind updating the location, to go alphabetically? Right now, it seems you would go after fyr.io. Thanks!